### ISSUE TITLE
Step 5 – songAtTrackNumber: should be songForTrackNumber:

### ISSUE THREAD

**Issue by kenmhaggerty** [learn-co-curriculum/jukebox-views#5]
Just a typo in step 5 of the lab instructions.

**Comment by markedwardmurray** [comment]
Oh sweet, good eyes. Do you know about the Edit tool in Learn? It's great for submitted fixes for typos, automatically creates a fork and PR with the change.

**Comment by kenmhaggerty** [comment]
Nope, but I can use it from now on if that is preferable, thanks! (Had been instructed to use "Raise an Issue" for typos / self-contained fixes.)
